packages/itwin/grouping-mapping-widget/src/components/Groups/GroupListItem.tsx

```diff
@@ -52,17 +52,24 @@ export const GroupListItem = ({
           {group.description}
         </ListItem.Description>
       </ListItem.Content>
-      {showGroupColor && overlappedElementsInfo.size > 0 &&
-        <OverlapProgress
-          group={group}
-          overlappedElementsInfo={overlappedElementsInfo}
-          groupElementsInfo={groupElementsInfo}
-        />}
-      <GroupMenuActions
-        group={group}
-        disableActions={disableActions}
-        {...rest}
-      />
+      <div className="gmw-group-list-end">
+        <div className="gmw-group-menu-actions">
+          <GroupMenuActions
```

Comment:
I see that the order of components changed. Could you provide before/after screenshots of how it looks? Or the OverlapProgress was not displayed at all in any case?

Reply:
I incorrectly ordered these components when I switched to the new list components. It was not displayed at all because of this. I must have forgotten to restore it to a working state when probably debugging something. This PR brings it back to how it was before the switch. 
Before (last working state):
![image](https://github.com/iTwin/viewer-components-react/assets/26608199/d5c2f729-9f65-40b4-8ad5-6939ba262500)
Now (with a slight adjustment to the spacing):
![image](https://github.com/iTwin/viewer-components-react/assets/26608199/990755b8-cde6-47aa-b4cd-508409db2e3d)